### PR TITLE
Functional tests for invalid image

### PR DESCRIPTION
**Pull request by machacekondra**
Signed-off-by: Ondra Machacek <omachace@redhat.com>

### CODE REVIEW

File: tests/ovirt/basic_vm_import_negative_test.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		err       error
+	)
+
+	BeforeEach(func() {
+		secret, err = f.CreateOvirtSecretFromBlueprint()
+		if err != nil {
+			Fail("Cannot create secret: " + err.Error())
+		}
+		namespace = f.Namespace.Name
+		err = f.OvirtStubbingClient.Reset("static-sso,static-transfer,static-namespace,static-transfers")
```
Comment (jakub-dzon) [inline tests/ovirt/basic_vm_import_negative_test.go]: It is already called in the common BeforeEach (https://github.com/kubevirt/vm-import-operator/blob/ec9095c98ca3b4c627ad68511871611508ea9cb0/tests/framework/framework.go#L176)

**Review by jakub-dzon — CHANGES_REQUESTED** [review]

**Review by jakub-dzon — APPROVED** [review]

**Review by pkliczewski — APPROVED** [review]

**Pull request merged**